scripts/travis_gdal_install.sh

```diff
@@ -32,7 +32,9 @@ GDALOPTS="  --with-ogr \
             --without-ingres \
             --without-xerces \
             --without-odbc \
-            --without-curl \
+            --with-curl \
+            --without-sqlite3 \
+            --with-curl \
```

Comment:
@youngpm will you remove the duplicate `--with-curl` and the `--without-sqlite3`?